runtime/async/async.go

```diff
@@ -45,11 +48,24 @@ func NewAsyncRuntime(port string, pattern string) (*Runtime, error) {
 			grpcHander: grpcHandler,
 		}, nil
 	}
-	handler, err := daprd.NewService(fmt.Sprintf(":%s", port))
+
+	var newService func(address string) (s dapr.Service, err error)
+	protocol := daprruntime.Protocol(os.Getenv(protocolEnvVar))
```

Comment:
why we are providing both HTTP and gRPC here?

Reply:
Just because developing dapr-proxy requires both grpc and http support (ff-nodejs using http).